modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/utils.py

```diff
@@ -20,22 +20,25 @@
 from functools import lru_cache
 from collections import OrderedDict
 
-import numpy as np
-
 import pandas
 from pandas import Timestamp
-from pandas.core.dtypes.common import get_dtype
+from pandas.core.dtypes.common import get_dtype, is_string_dtype
 from pandas.core.arrays.arrow.extension_types import ArrowIntervalType
 
 import pyarrow as pa
 from pyarrow.types import is_dictionary
 
+import numpy as np
+
 from modin.utils import MODIN_UNNAMED_SERIES_LABEL
 
+_EMPTY_ARROW_TABLE = None
```

Comment:
Why not? Instead of using `empty_arrow_table`.
```suggestion
_EMPTY_ARROW_TABLE = pa.Table.from_pandas(pandas.DataFrame({}))
```

Reply:
Just to save memory a little bit. This empty table may never be used at runtime or used quite rarely, thus, it's created lazy, on the first access.

Reply:
What is the amount of memory you want to save? If it is not significant, then let's not complicate the code.

Reply:
No, it's not significant.